prow/jenkins/controller_test.go

```diff
@@ -63,13 +63,15 @@ func newFakeConfigAgent(t *testing.T, maxConcurrency int, operators []config.Jen
 			},
 			RunAfterSuccess: []config.Presubmit{
 				{
+					AlwaysRun: true,
```

Comment:
Technically we had the same issue with RunAfterSuccess jobs as we did with Gerrit -- `run_always: false` was ignored and effectively defaulted to `true`.